e2e/pod.go

```diff
@@ -13,10 +14,15 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+const (
+	hostNetworkDebugPodNamePrefix    = "debug"
+	nonHostNetworkDebugPodNamePrefix = "debugnonhost"
```

Comment:
is it a `prefix` or `app: label` value?

Reply:
good point, its the name of our daemonset. Will adjust the name